app/services/smt.js

```diff
@@ -410,9 +407,7 @@ export default Service.extend({
 
     space.get('channels').removeObject(channel);
 
-    // Update persisted channel list
-    this.get('storage.rs').kosmos.spaces.store(space.serialize())
-      .then(() => this.log('leave', 'stored space', space.get('name')));
```

Comment:
It's pretty much the same log message as the `saveSpace` method of the `storage` service logs as well. And the rest of the code is identical as well, that's why I replaced it.